x-pack/plugins/apm/server/routes/services/get_services/get_service_aggregated_transaction_stats.ts

```diff
@@ -91,11 +91,6 @@ export async function getServiceAggregatedTransactionStats({
                           field: TRANSACTION_DURATION_SUMMARY,
                         },
```

Comment:
nit: the file is a bit confusing. We have transaction metrics and service metrics. "service aggregated transaction" mean aggregated transactions (aka transaction metrics) or does it mean service metrics?

Reply:
In the past we referred to transaction metrics as "aggregated transactions". I suggest we standardize on "transaction metrics" and "service metrics".

Reply:
I agree the naming is **so** confusing. (at least for me) 
@sqren  let's move the discussion [here](https://github.com/elastic/kibana/issues/141185#issuecomment-1254676875) and agree on the naming